### DISCUSSION TITLE
Let the Discussions begin!

### DISCUSSION THREAD

**Discussion by chrisdutz** [apache/plc4x#946] [General]
We now have enabled GitHub Discussions ... 

However I will be using this thread to see if the email subject settings are working as expected :-)

**Comment by sruehl** [comment]
are they?

**Comment by chrisdutz** [comment]
Well I asked Infra why they are not appearing ... they said, they finished setting things up for us (You needed to open a ticket for GH Discussions)

**Comment by chrisdutz** [comment]
LOL ... well I copied the row in our .asf.yaml from the streampipes folks and didn't update it to send to dev@plc4x.apache.org .... :picard-facepalm: (you can imagine what the emoji would be)

**Comment by chrisdutz** [comment]
As the Germans say: "Kaum macht man's richtig, gehts!" ;-) (Once you start doing it right, it starts working)
